service/src/main/java/org/apache/celeborn/server/common/service/config/ConfigService.java

```diff
@@ -32,6 +32,22 @@ default DynamicConfig getTenantConfig(String tenantId) {
     }
   }
 
+  UserConfig getRawUserConfig(String tenantId, String userId);
```

Comment:
`UserIdentifier` is consist of `tenantId` and `name`, therefore we could extend `getTenantConfig` like `getTenantConfig(String tenantId, String name)`.

Reply:
One tenant may have many user, put it in one Config is necessary?

Reply:
The `tenantId` is required and the `name` is optional. Therefore the `getTenantConfig` returns the tenant config given `tenantId` and null `name`, and the tenant user configs merged with the tenant config given `tenantId` and `name` of user.

Reply:
> @AngersZhuuuu, the `tenantId` is required and the `name` is optional. Therefore the `getTenantConfig` returns the tenant config given `tenantId` and null `name`, and the tenant user configs merged with the tenant config given `tenantId` and `name` of user.

Prefer use  `getRawTenantUserConfig` for just tenant-user config